test/unit/common/auth/authentication-events.run.spec.js

```diff
@@ -11,9 +12,16 @@ describe('global event handlers', function () {
         };
 
     beforeEach(function () {
-
         var testApp = makeTestApp();
-
+        mockedPostFiltersService = {
```

Comment:
@crcommons  just a silly test mock because otherwise the auth events failed due to not having the resetDefaults & PostFilters available =)